src/Evolution/Systems/GrMhd/ValenciaDivClean/ConservativeFromPrimitive.cpp

```diff
@@ -70,16 +72,20 @@ void ConservativeFromPrimitive::apply(
   get(*tilde_d) = get(sqrt_det_spatial_metric) * get(rest_mass_density) *
                   get(lorentz_factor);
 
-  get(*tilde_tau) = get(sqrt_det_spatial_metric) *
-                      (square(get(lorentz_factor)) *
-                        (get(rest_mass_density) *
-                          (get(specific_internal_energy) +
-                           get(spatial_velocity_squared) * get(lorentz_factor) /
-                              (get(lorentz_factor) + 1.)) +
-                         get(pressure) * get(spatial_velocity_squared)) +
-                       0.5 * get(magnetic_field_squared) *
-                        (1.0 + get(spatial_velocity_squared)) -
-                       0.5 * square(get(magnetic_field_dot_spatial_velocity)));
+  get(*tilde_ye) = get(sqrt_det_spatial_metric) * get(rest_mass_density) *
```

Comment:
just multiply tilde_d and electron_fraction

Reply:
Ok. Done.